packages/jbrowse-plugin-apollo/src/components/ImportFeatures.tsx

```diff
@@ -131,16 +203,46 @@ export function ImportFeatures({
               </MenuItem>
             ))}
           </Select>
+        </DialogContent>
+        <DialogContent style={{ display: 'flex', flexDirection: 'column' }}>
           <DialogContentText>Upload GFF3 to load features</DialogContentText>
           <input
             type="file"
             onChange={handleChangeFile}
             disabled={submitted && !errorMessage}
           />
         </DialogContent>
+
+        {featuresCount > 0 ? (
+          <DialogContent>
+            <DialogContentText>
+              This assembly already has {featuresCount} features, would you like
+              to delete the existing features before importing new ones?
+            </DialogContentText>
+            <FormControlLabel
+              label={
+                deleteFeatures
+                  ? 'Yes, delete existing features'
+                  : 'No, import without deleting existing features'
```

Comment:
The label text should not change when the checkbox is clicked, just keep it as 'Yes, delete existing features'. Also, the default should be unchecked.